### DISCUSSION TITLE
How to disable/remove statusline?

### DISCUSSION THREAD

**Discussion by tmpm697** [helix-editor/helix#7934] [Q&A]
I use text editor in conjunction with other software like multiplexing, having a fixed horizontal line in a window/pane is bad for intergrations.

So is there a way to disable / remove statusline bar in helix and if not, does any ppl come to the same use-case like me?

**Comment by nik-rev** [comment]
I've made a PR which allows the statusline to be merged with the command line, which by extension means the statusline can be disabled if you set all of the `left`, `center` and `right` components to an empty array

See https://github.com/helix-editor/helix/pull/12204

**Reply by kaem-e** [reply]
this would be awesome to see i personally never use the statusline so its quite annoying having it always be present
